controllers/datafoundation.go

```diff
@@ -214,6 +244,49 @@ func (r *dataFoundationReconciler) setDeviceSetCount(deviceSet *ocsv1.StorageDev
 		deviceSet.Count = currDeviceSetCount
 	}
 }
+
+func (r *dataFoundationReconciler) reconcileCSV() error {
+	r.Log.Info("Reconciling OCS Operator CSV")
+	csvList := opv1a1.ClusterServiceVersionList{}
+	if err := r.list(&csvList, &client.ListOptions{
+		Namespace: r.dfNamespace,
+	}); err != nil {
+		return fmt.Errorf("unable to list csv resources: %v", err)
+	}
+	var csv *opv1a1.ClusterServiceVersion
+	for index := range csvList.Items {
+		obj := &csvList.Items[index]
+		if strings.HasPrefix(obj.Name, ocsOperatorName) {
+			csv = obj
+			break
+		}
+	}
+	if csv == nil {
+		r.Log.Info("unable to find OCS CSV")
+		return nil
+	}
+
+	isChanged := r.addOwnerAnnotation(csv)
+	deployments := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
+	for i := range deployments {
+		containers := deployments[i].Spec.Template.Spec.Containers
+		for j := range containers {
+			container := &containers[j]
+			resources := datafoundation.GetResourceRequirements(container.Name)
+			if !equality.Semantic.DeepEqual(container.Resources, resources) {
+				container.Resources = resources
+				isChanged = true
+			}
```

Comment:
What happened to the warning that was here?

Reply:
Updated